src/xunit.analyzers/Descriptors.cs

```diff
@@ -254,7 +254,10 @@ internal static class Descriptors
             "Keep precision in range {0} when asserting equality of {1} typed actual value.",
             Categories.Assertions, DiagnosticSeverity.Error, isEnabledByDefault: true);
 
-        // Placeholder for rule X2017
+        internal static DiagnosticDescriptor X2017_AssertIsTypeShouldNotBeUsedForAbstractType { get; } = new DiagnosticDescriptor("xUnit2017",
```

Comment:
Yes, but you still need to coordinate the analyzer ID numbering. Without a centralized place it would be too easy to have multiple rules use the same id, potentially confusing fixers, ignore pragmas, etc. Plus at the moment this file servers as a listing of all the available rules until I get around to creating some real documentation. But even then this central list would likely remain.